bootstrap.sh

```diff
@@ -11,8 +11,8 @@ function checkout() {
 stage_1_dir=./bootstrap-1
 stage_2_dir=./bootstrap-2
 
-checkout bootstrap/2 $stage_2_dir
 checkout bootstrap/1 $stage_1_dir
+checkout bootstrap/2 $stage_2_dir
```

Comment:
that did bother me ngl.. but I didn't want to be tooo annoying :)

Reply:
:stuck_out_tongue_closed_eyes: